<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>明文</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="plaintextComboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPlainTextEdit" name="plaintext"/>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextFormatShow"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>密钥</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>密钥剩余长度</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="privateKeyLen">
              <property name="text">
               <string>56</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stringKeyGer">
              <property name="text">
               <string>随机生成密钥</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPlainTextEdit" name="private_key"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>密文</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPlainTextEdit" name="ciphertext"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>明文</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="plaintextComboBox_1"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plaintext_1"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="encrypt">
              <property name="text">
               <string>加密</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="decrypt">
              <property name="text">
               <string>解密</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clear">
              <property name="text">
               <string>清空</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>消息</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="logsSave">
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="logs"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>选取文件：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inFileSelectedPath"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectInFile">
              <property name="text">
               <string>选取文件</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>解/加密</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="opComboBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>速度</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="speedComboBox"/>
            </item>
            <item>
             <widget class="QCheckBox" name="decryptValid">
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>解密校验</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>密钥:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="filePrivateKey"/>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>密钥剩余长度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="residueKey">
              <property name="text">
               <string>56</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fileKeyGer">
              <property name="text">
               <string>生成随机密钥</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>输出文件：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="outFileSelectedPath"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectOutFile">
              <property name="text">
               <string>选取文件</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>完成比例</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="rate">
              <property name="text">
               <string>0%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QPushButton" name="start">
              <property name="text">
               <string>开始</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pause">
              <property name="text">
               <string>终止</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
